frontend/src/dispatcher/js-dispatcher.ts

```diff
@@ -30,26 +31,26 @@ export function createJsDispatcher() {
 			return;
 		}
 
+		// TODO: Provide an explanatory comment here
```

Comment:
// Checks if the provided `messageConstructor` is a class extending `JsMessage`. All classes inheriting from `JsMessage` will have a static readonly `jsMessageMarker` which is true.